### ISSUE TITLE
OWL works, but opendrop doesn't

### ISSUE THREAD

**Issue by littleblack111** [seemoo-lab/owl#88]
for context: I’m using Intel Wireless 7265 for wifi card, but the network is using ethernet
I am using https://github.com/seemoo-lab/owl and https://github.com/seemoo-lab/opendrop. I use owl -i wlp2s0 -N(because without -N, ERROR: Error while receiving via netlink: Operation not supported), then opendrop recive. doesn't work, opendrop find doesn't work as well. although I can see when i click airdrop on macbook, it will add peer on owl's log

**Comment by Wildzekat** [comment]
@littleblack111 
-N tells it to not convert by itself but use existing monitor devices *or other methods* like Nexmon and thus will not work if you did not set a card into active monitor mode by yourself.

I assume you ran it as monitor, but that doesnt do the job completely. You need **active** monitor mode for that.

**Comment by littleblack111** [comment]
> @littleblack111 
> -N tells it to not convert by itself but use existing monitor devices *or other methods* like Nexmon and thus will not work if you did not set a card into active monitor mode by yourself.
> 
> I assume you ran it as monitor, but that doesnt do the job completely. You need **active** monitor mode for that.

Yep, figured that my card don't support *active* monitoring mode
